docker-compose/cse-import-ec-d2cc/config/cse-d2cc-gridcapa-task-manager-application.yml

```diff
@@ -5,6 +5,10 @@ spring:
     username: cse_import_ec_d2cc_server
     password: cse-import-ec-d2cc
   jpa:
+    properties:
```

Comment:
Why didn't you add the same changes to CSE Import EC IDCC, CSE Import IDCC and SWE IDCC IDCF?